PySDM/backends/impl_numba/methods/seeding_methods.py

```diff
@@ -0,0 +1,50 @@
+""" CPU implementation of backend methods for particle injections """
+
+from functools import cached_property
+
+import numba
+
+from PySDM.backends.impl_common.backend_methods import BackendMethods
+
+
+class SeedingMethods(BackendMethods):  # pylint: disable=too-few-public-methods
+    @cached_property
+    def _seeding(self):
+        @numba.njit(**{**self.default_jit_flags, "parallel": False})
+        def body(
+            idx,
+            multiplicity,
+            extensive_attributes,
+            seeded_particle_multiplicity,
+            seeded_particle_extensive_attributes,
+        ):
+            for i, mult in enumerate(multiplicity):
+                if mult == 0:
+                    idx[i] = -1
+                    multiplicity[i] = seeded_particle_multiplicity
+                    for a in range(len(extensive_attributes)):
+                        extensive_attributes[a, i] = (
```

Comment:
Re: my comment in https://github.com/open-atmos/PySDM/issues/1331 -- I think it would be great instead of just setting the extensive attributes to fixed values here to instead set them each time to a value that is sampled from the seeded particle distribution.

Reply:
here's a first try at a sampling logic: https://github.com/open-atmos/PySDM/pull/1367/commits/c17f338ad2ff5a6b4a8b62f9033b75e8f43aa6d8